src/Test/contconic.jl

```diff
@@ -1428,6 +1437,10 @@ function _det1test(model::MOI.ModelLike, config::TestConfig, vecofvars::Bool, de
 
         @test MOI.get(model, MOI.VariablePrimal(), t) ≈ expectedobjval atol=atol rtol=rtol
 
+        if logdet
+            @test MOI.get(model, MOI.VariablePrimal(), u) ≈ 1. atol=atol rtol=rtol
```

Comment:
The JuMP style guide prefers `1.0` over `1.`

Reply:
I was following the style in surrounding code. I can do a PR for style consistency throughout moi after informs next week. If there's one thing I'm good at, it's being anal.

Reply:
Yes, the surrounding code was written pre-JuMP style guide and should be updated. I agree that it fits a separate PR